navigating-the-single-cell-landscape-a-comprehensive-workshop-on-scrnaseq/day-1/4-psuedobulk.qmd

```diff
@@ -17,208 +17,532 @@ editor_options:
 
```

Comment:
Add this chunk to set options for all chunks below 
```suggestion
```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE, out.width = "100%", fig.align='center', 
                      message = FALSE, warning = FALSE, cache = FALSE)
options(width = 1200)
```
```